rust/face-recognition/upload-models-to-canister.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+dfx canister call backend clear_face_detection_model_bytes
+dfx canister call backend clear_face_recognition_model_bytes
+upload_byte_file backend append_face_detection_model_bytes . version-RFB-320.onnx
+upload_byte_file backend append_face_recognition_model_bytes . face-recognition.onnx
```

Comment:
When I compiled `upload_byte_file` from master, the command failed with the following error:

```
Error: "Not enough arguments. Usage: <program> <canister_name> <canister_method_name> <model_folder_path> <models_list> <start_ind> [network_type]"
```

I believe this is because the version on master has added an additional required argument - the starting index. The following change fixes it:

```suggestion
upload_byte_file backend append_face_detection_model_bytes . version-RFB-320.onnx 0
upload_byte_file backend append_face_recognition_model_bytes . face-recognition.onnx 0
```

Perhaps it would be good to add an instruction to checkout commit `d3d2bf2d184c711e15d2b18a6af27aa56484c640` in the `rust-connect-py-ai-to-ic` repo so that the demo continues to work even if the command changes later in master?

Reply:
Thanks for review, Islam!

I updated the code with a new `ic-file-uploader` tool that Jeshli implemented based of `upload_byte_file`. The code should work now.